oomph-plugin-notes.js

```diff
@@ -0,0 +1,76 @@
+(function($) {
+	$('.plugin-notes-container.editable, .plugin-notes-edit').on('click', function() {
+		var $container = $(this).closest('.plugin-notes-container')
+		  , $textarea = $('<textarea class="widefat">')
+		  , $saveButton = $('<button class="button-primary">')
+		  , $cancelButton = $('<button class="button-secondary">')
+		;
+
+		if($container.hasClass('editing')) {
+			return;
+		}
+
+		$container.addClass('editing');
+
+		$saveButton.text(OPN.text.save);
+		$cancelButton.text(OPN.text.cancel);
+
+		$container.data('plugin-markup', $container.html());
+
+		$textarea.val($container.data('plugin-notes'));
+
+		function adjust() {
+			$textarea.height(1);
```

Comment:
It's necessary because `scrollHeight >= clientHeight`, always; so even when there's LESS text in there, the box would grow in height otherwise since the `clientHeight` is already overridden. Setting the height to 1 first resets it and thus `scrollHeight` indicates just the height of the text.